extension-manifest-v3/src/background/adblocker.js

```diff
@@ -84,11 +84,9 @@ const setup = asyncSetup([
 
     // Clean previous regional engines
     if (lastValue) {
-      await Promise.all(
-        lastValue.regions
-          .filter((id) => !regions.includes(id))
-          .map((id) => engines.clean(`lang-${id}`)),
-      );
+      lastValue.regions
```

Comment:
It's leftover, as the `engines.clean()` is not async function (it was initially).